lib/license.js

```diff
@@ -153,3 +220,28 @@ function writeTemplatedLicense(newName, oldName, text, pkgName) {
   } catch(_) {
   }
 }
+
+function hasOthers() {
+  var old = readLicense();
+  return old && /strongloop.*other.*contributors/i.test(old);
```

Comment:
I don't think I would actually remove the local variable like in my example, mind you.